packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -296,6 +296,14 @@ const registerNetworkIpc = (mainWindow) => {
       /** @type {import('axios').AxiosResponse} */
       const response = await axiosInstance(request);
 
+      const dataBuffer = Buffer.from(response.data);
+      // Overwrite the original data for backwards compatability
+      response.data = dataBuffer.toString('utf-8');
```

Comment:
What about using charset from "Content-Type" with a fallback to UTF-8? Something like this:
```js
      const content_type = response.headers['content-type'];
      // RegExp magic from https://stackoverflow.com/a/33192813
      let charset = new RegExp('/charset=([^()<>@,;:\"/[\]?.=\s]*)/i').exec(content_type);
      if (charset == null) {
        charset = "utf-8"
```